tests/test_val.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+import val
+
+
+def test_run():
+    configs = {"data": val.ROOT / "data/coco128.yaml", "imgsz": 128, "weights": "yolov5n.pt"}
+    _mP = 0.36729776579718126
+    _mR = 0.24184124611503316
+    _mAP50 = 0.2105258344273701
+    _mAP = 0.11256317077174217
+
+    (mP, mR, map50, mAP, *loss), mAPs, t = val.run(**configs)
+    assert np.isclose(mP, _mP, rtol=1e-3)
+    assert np.isclose(mR, _mR, rtol=1e-3)
+    assert np.isclose(map50, _mAP50, rtol=1e-3)
+    assert np.isclose(mAP, _mAP, rtol=1e-3)
```

Comment:
consider to place the hard numbers here directly

Reply:
updated